podvm/README.md

```diff
@@ -73,48 +74,82 @@ $ docker build -t podvm_builder \
+Assuming you have built the podvm_builder image for Ubuntu as explained in the previous step,
+running the following command will build the image with the podvm binaries.
 
-The builder image has to be indicated via `BUILDER_IMG` build argument to docker. Apart from that, the `CLOUD_PROVIDER` argument is mandatory and specifies the cloud provider that binaries should be built for. Once again, one builder image can be used to build for multiple
-cloud providers and architectures.
+```
+$ docker build -t podvm_binaries \
+         --build-arg BUILDER_IMG=podvm_builder \
+	 -f Dockerfile.podvm_binaries .
 
-Below is shown how to build an Ubuntu image for libvirt:
+```
+The build process can take significant time.
+
+## Building the podvm qcow2 image
+
+In order to build the podvm image you should be using the corresponding
+dockerfile of the Linux distro for which the builder and binaries image were
+built.  For example, if the builder image was built with
+*Dockerfile.podvm_builder.DISTRO* then you should use the
+*Dockerfile.podvm.DISTRO* to build the podvm image.
+
+The builder image has to be indicated via `BUILDER_IMG` build argument and
+binaries image has to be indicated via `BINARIES_IMG` build argument to docker.
+
+Below command will build the qcow2 image that can be used for all cloud providers
+based on Ubuntu distro.
 
 ```
-$ docker build -t podvm_libvirt \
-	--build-arg BUILDER_IMG=podvm_builder:latest \
-	--build-arg CLOUD_PROVIDER=libvirt \
-	-f Dockerfile.podvm .
+$ docker build -t podvm \
+         --build-arg BUILDER_IMG=podvm_builder \
+	 --build-arg BINARIES_IMG=podvm_binaries \
+	 -f Dockerfile.podvm .
 ```
 
-This step will take several minutes to complete, mostly because `packer` will use the QEMU builder in emulation mode when running within container. If you are using podman then you can speed up QEMU by enabling native virtualization, i.e., passing the `--device=/dev/kvm` argument to enable KVM inside the container.
+This step will take several minutes to complete, mostly because `packer` will
+use the QEMU builder in emulation mode when running within container. If you
+are using podman then you can speed up QEMU by enabling native virtualization,
+i.e., passing the `--device=/dev/kvm` argument to enable KVM inside the
+container.
 
-Also beware that the process consume a bunch of memory and disk from the host. As a tip, if the build fail at the point QEMU was launched but packer couldn't connect via ssh then it might indicate lack of memory (try to increase the amount of memory if running on VM).
+Also beware that the process consume a bunch of memory and disk from the host.
+As a tip, if the build fail at the point QEMU was launched but packer couldn't
+connect via ssh then it might indicate lack of memory (try to increase the
+amount of memory if running on VM).
```

Comment:
Nit: formatting and adding example error
```suggestion
> **Note:** Beware that the process consume a bunch of memory and disk from the host.
If the build fails at the point QEMU was launched but packer couldn't
connect via ssh, with an error similar to:
> ```
> Build 'qemu.ubuntu' errored after 5 minutes 57 seconds: Timeout waiting for SSH.
> ```
> then it might indicate lack of memory, so try to increase the amount of memory if running on VM.
```